API_CHANGELOG.md

```diff
@@ -14,3 +14,7 @@ Synapse trimming was an optimization which is no longer possible because of an i
 
 * Removed method SpatialPooler::setSynPermMax as the maximum permanence is now defined (hardcoded) as
 nupic::algorithms::connections::maxPermancence = 1.0f;
+
+* Changed callback ConnectionsEventHandler::onUpdateSynapsePermanence().  Instead of being called
+every time a synapses permanence changes, it is not called when a synapse changes connected state,
```

Comment:
"it is not called when" ->  "it is now called only when"  ?

Reply:
Thanks for the continuing feedback @breznak !  I will attend to this within the next day.

Reply:
I'm looking forward to have this merged soon!